examples/workflow/pygrb/injections.ini

```diff
@@ -0,0 +1,121 @@
+[executables]
```

Comment:
Looking at some minor details here.  In the [injections-detectionfullinj] section
max-mass1    = 25
max-mass2     = 3
but
max-mtotal   = 25
Should this be raised to 28?  Also
max-spin1     = 0.98
max-spin2     = 0.75
are different: make sure this is what we want (perhaps this should raised on a GRB call).  Same for all the NS max spins in [injections-nsns30inj] and [injections-nsbh30inj] which are set to 0.40.